db/migrate/20150930183738_migrate_content_hosts.rb

```diff
@@ -1,14 +1,19 @@
 class MigrateContentHosts < ActiveRecord::Migration
-  class Katello::HostCollectionAssociation < ActiveRecord::Base
+  class Katello::SystemRepository < ActiveRecord::Base
```

Comment:
If I do that what happens when I call code from my migration that in turn calls `Katello::Repository.some_method()`?